javascript/ql/src/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -108,6 +108,12 @@ module TaintedPath {
         not (isNormalized() and isAbsolute())
       }
     }
+
+    class SplitPath extends DataFlow::FlowLabel {
```

Comment:
qldoc

It would be nice to mention that this represents an array containing `..` elements.